### PR TITLE
Mobile 4499

### PR DESCRIPTION
**Pull request by dpalou**

### CODE REVIEW

File: src/addons/calendar/pages/event/event.ts [hunk 1]
```diff
@@ -61,6 +64,7 @@ export class AddonCalendarEventPage implements OnInit, OnDestroy {
     protected defaultTimeChangedObserver: CoreEventObserver;
     protected currentSiteId: string;
     protected updateCurrentTime?: number;
+    protected appResumeSubscription: Subscription;
```
Comment (crazyserver) [inline src/addons/calendar/pages/event/event.ts:67]: Just a personal thing: Add subscriptions array and observers array to then turn them off altogether on destroy.

File: src/addons/notifications/pages/list/list.ts [hunk 2]
```diff
@@ -47,12 +51,15 @@ export class AddonNotificationsListPage implements AfterViewInit, OnDestroy {
...
     loadingMarkAllNotificationsAsRead = false;
+    hasNotificationsPermission = true;
+    permissionWarningHidden = false;
 
     protected isCurrentView?: boolean;
     protected cronObserver?: CoreEventObserver;
     protected readObserver?: CoreEventObserver;
     protected pushObserver?: Subscription;
     protected pendingRefresh = false;
+    protected appResumeSubscription?: Subscription;
```
Comment (crazyserver) [inline src/addons/notifications/pages/list/list.ts:62]: Same (array thing)

File: src/theme/theme.base.scss [hunk 3]
```diff
@@ -931,6 +931,25 @@ ion-card {
...
+
+    &.core-card-with-buttons .item ion-label {
+        margin-bottom: 0;
+    }
+
+    .core-card-buttons {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        @include margin(0, 8px, 8px, 8px);
```
Comment (crazyserver) [inline src/theme/theme.base.scss:943]: Since start and end (left and right) is the same, use margin: directly without the include.

**Review by crazyserver — CHANGES_REQUESTED** [review]

File: src/theme/theme.base.scss (hunk 3 above)
Comment (dpalou) [reply src/theme/theme.base.scss]: Changed (the CSS was included in the previous issue though, but I fixed it here).

**Review by crazyserver — APPROVED** [review]

**Pull request merged**